prefect_dbt/cli/tasks.py

```diff
@@ -0,0 +1,129 @@
+import os
+from pathlib import Path
+from typing import Optional, Union
+
+import yaml
+from dbt.cli.main import dbtRunner, dbtRunnerResult
+from dbt.contracts.results import NodeStatus
+from prefect import get_run_logger, task
+from prefect.artifacts import create_markdown_artifact
+from pydantic import VERSION as PYDANTIC_VERSION
+
+from prefect_dbt.cli.credentials import DbtCliProfile
+
+if PYDANTIC_VERSION.startswith("2."):
+    pass
+else:
+    pass
+
+
+@task
+def dbt_build_task(
+    profiles_dir: Optional[Union[Path, str]] = None,
+    project_dir: Optional[Union[Path, str]] = None,
+    overwrite_profiles: bool = False,
+    dbt_cli_profile: Optional[DbtCliProfile] = None,
+    dbt_client: Optional[dbtRunner] = None,
+    create_artifact: bool = True,
+    artifact_key: str = "dbt-build-task-summary",
```

Comment:
would it make sense to parameterize this for a given task so the user doesn't always overwrite the same key by default?

Reply:
I am going to leave this as the default for the dbt build command because dbt build runs all of the models/tests/snapshots/seeds/etc. We will have other pre-built tasks with their own artifacts for each of the individual node types that I will make sure have unique keys for the artifacts